app/routes/routes.py

```diff
@@ -113,15 +118,17 @@ def get_question(questionId):
                 return jsonify(temp), 200
```

Comment:
the returned json should come with the all answers provided so far for the question according to the notes in the doc.

Reply:
Thanks, I implemented that in a new connection branch, about to send a PR